client/header/activity-panel/panels/inbox/card.js

```diff
@@ -256,24 +251,16 @@ class InboxNoteCard extends Component {
 			if ( ! noteActions ) {
 				return [];
 			}
-			return noteActions.map( ( action ) => (
-				<NoteAction
-					key={ noteId }
-					noteId={ noteId }
-					action={ action }
-				/>
+			return noteActions.map( ( action, index ) => (
+				<NoteAction key={ index } noteId={ noteId } action={ action } />
 			) );
 		};
 
 		const actionsList = getButtonsFromActions( note );
 
 		return (
 			actionsList && (
-				<Fragment>
-					{ actionsList.map( ( item, i ) =>
-						cloneElement( item, { key: i } )
-					) }
-				</Fragment>
+				<Fragment>{ actionsList.map( ( action ) => action ) }</Fragment>
```

Comment:
The `.map()` is unnecessary here.

Reply:
Nice catch! I simplified the function a little bit in the commit 30bc1ab